app/assets/javascripts/admin/edition_publishing.js

```diff
@@ -217,3 +217,21 @@ jQuery(function($) {
 
   SetTopicsFromPolicy.init();
 }(jQuery));
+
+(function($) {
+  var $policy_input = $('#edition_policy_content_ids'),
+      $important_checkbox = $('#edition_important');
```

Comment:
Can these two also be updated to `$policyInput` and `$importantCheckbox` respectively.